src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,57 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Line {
+
+    private static Random random = new Random();
+
+    // Line 은 라인사이의 있는 상태를 뜻하므로 -1 을 해줌
+    private static int SUBTRACT_NUMBER = 1;
+
+    /*
+     * true가 사다리 사이 이어짐
+     * false가 이어지지 않음
+     * ex) List에 담긴 값이 차례대로 true, false, true 일 때 아래와 같이 존재
+     * |----|    |----|
+     */
+    private List<Boolean> points = new ArrayList<>();
+
+    public Line(final int countOfPerson) {
+        initLineList(countOfPerson);
+    }
+
+    private void initLineList(final int countOfPerson) {
+
+        for (int i = 0; i < countOfPerson - SUBTRACT_NUMBER; i++) {
+            addPoint();
+        }
+    }
+
+    private void addPoint() {
+        if (points.size() == 0) {
+            points.add(random.nextBoolean());
+
+            return;
+        }
+
+        checkPointRepeat();
+    }
+    
+    private void checkPointRepeat(){
```

Comment:
메소드 네이밍이 어색해보입니다. 조금더 의미있는 메소드명을 고려보세요 ! 
```suggestion
    private void checkPointRepeat() {
```
메소드의 중괄호 사이는 공백을 추가해주는것이 컨벤션입니다.